doc/doxygen/headers/glossary.h

```diff
@@ -327,14 +327,10 @@
  * this, here setting the boundary indicator to 42 for all faces located at
  * $x=-1$:
  * @code
- *   for (typename Triangulation<dim>::active_cell_iterator
- *          cell = triangulation.begin_active();
- *        cell != triangulation.end();
- *        ++cell)
- *     for (unsigned int f=0; f<GeometryInfo<dim>::faces_per_cell; ++f)
- *       if (cell->face(f)->at_boundary())
- *         if (cell->face(f)->center()[0] == -1)
- *           cell->face(f)->set_boundary_id (42);
+ *   for ( auto &face : triangulation.active_face_iterators() )
```

Comment:
Can you remove the whitespace?
```suggestion
 *   for (auto &face : triangulation.active_face_iterators())
```